src/components/homePage/Testimonials.vue

```diff
@@ -1,19 +1,236 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <div>
+      <img src="../../assets/testimonial_building.svg" id="building" />
+      <vueper-slides 
+        class="no-shadow" 
+        :bullets="false" 
+        :arrows="false" 
+        :touchable="false" 
+        autoplay
+      >
+        <vueper-slide 
+          v-for="(slide, i) in information['slides']" 
+          :key="i"
+        >
+          <template v-slot:content>
+            <h2 id="hear"><strong>Hear what people have to say about CUTC!</strong></h2>
```

Comment:
Is there a reason why this is wrapped in an h2 AND a strong? I think I'd prefer using a bold font within the class here